tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/readonly/ImmutabilityBalaTest.java

```diff
@@ -96,6 +100,7 @@ public void testImmutableTypesNegative() {
                 ":Config & readonly)'", 85, 5);
         validateError(result, index++, "cannot update 'readonly' value of type 'testorg/selectively_immutable:1.0.0" +
                 ":MyConfig'", 88, 5);
+        validateError(result, index++, "missing required parameter '' in call to 'utcToCivil()'", 92, 18);
```

Comment:
There seems to be a TODO related to the param name not being set - https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java#L537. Will create an issue.